ophyd/signal.py

```diff
@@ -726,6 +773,9 @@ def set_default_timeout(cls, *, timeout=2.0, connection_timeout=1.0):
         cls.__default_connection_timeout = connection_timeout
         cls.__default_timeout = timeout
 
+    # TODO Is there a good reason to prohibit setting these three timeout
```

Comment:
Is there a good reason to allow the user to change them on the fly? 🤷‍♂

Reply:
Yeah, it might be convenient to be able to try different write timeouts, but if (per the docstring) the recommended approach is to configure this inside a `set` or `trigger`, where we have the most information about how long an action should take, then this is a moot point.